modules/caddyhttp/replacer.go

```diff
@@ -119,6 +119,20 @@ func addHTTPVarsToReplacer(repl *caddy.Replacer, req *http.Request, w http.Respo
 				return port, true
 			case "http.request.hostport":
 				return req.Host, true
+			case "http.request.local":
+				localAddr, _ := req.Context().Value(http.LocalAddrContextKey).(net.Addr)
+				return localAddr.String(), true
+			case "http.request.local.host":
```

Comment:
Why not use the same logic as `http.request.remote.host`? Suppose local address is a unix socket, this will return an empty string.

Reply:
Ah, so that's why `http.request.remote.host` is the way it is!

Thanks!

I consided that a dead code path and didn't think of unix sockets.

Will add comments to mention unix sockets to `http.request.remote.host` and `http.request.local.host` together with the fix tomorrow or so :)

Reply:
> Will add comments to mention unix sockets to `http.request.remote.host` and `http.request.local.host` together with the fix tomorrow or so :)

done